iset.mm

```diff
@@ -56423,7 +56423,7 @@ associative structure (such as a group).  (Contributed by NM,
   the second argument is the first component of the first argument, in short
   "x-maps-to operations".  For labels, the abbreviations "mpt2x" are used
   (since "x" usually denotes the first argument).  This is in line with the
-  currently used conventions for such cases (see ~ cbvmpox , ~ ovmpt2x and
+  currently used conventions for such cases (see ~ cbvmpox , ~ ovmpox and
   ~ fmpt2x ).  However, we are in the process of renaming to use the
   abbreviation "mpo" instead of "mpt2". As this proposal is realized, the
   labels in the following should also be adapted. If the first argument is
```

Comment:
Please don#t forget to revise this comment after finishing the replacements.

Reply:
Good point, I'll add a checklist in the GitHub issue

Reply:
I have added the checklist to the top of https://github.com/metamath/set.mm/issues/98 . Please edit that issue or let me know if there is anything I missed.